Source/Lib/Codec/EbResourceCoordinationProcess.c

```diff
@@ -472,7 +472,7 @@ void* ResourceCoordinationKernel(void *inputPtr)
         // Parent PCS is released by the Rate Control after passing through MDC->MD->ENCDEC->Packetization
         EbObjectIncLiveCount(
             pictureControlSetWrapperPtr,
-            1);
```

Comment:
You mean in some case, pictureControlSetWrapperPtr is released between here and line 614?
And at 614 `EbObjectIncLiveCount(&p, 1); //bug, increase live count after release` is where bug shows up?

Reply:
yes

Reply:
Thanks. I see. Looks fine and should not hurt anything. But I need some more time to run some tests and make sure this PR doesn't bring regression to the issue fixed by https://github.com/OpenVisualCloud/SVT-HEVC/commit/b2530415115efeb492923cc3d0604cf2b070a9b9.

Reply:
my change EbResourceCoordinationProcess.c should be safe to merge, the impact is small

but my change in EbSystemResourceManager.c has bigger impact. i can see the warning at line 396 and line 609 is printed in my project. need further investigation for the multi thread issue.   no hurry to merge.